modules/web-mercator/docs/api-reference/web-mercator-viewport.md

```diff
@@ -91,6 +91,19 @@ Returns:
  - `[longitude, latitude]`
 
 
+##### `getBounds(options)`
+
+Returns the bounding box of the current visible area.
+
+* `options` (Object, optional)
+  + `options.z` (Number, optional) - The altitude at which the bounds should be calculated. Default `0`.
```

Comment:
I am not sure what the use case for this is. Should this be seen as the max altitude of the data that is being displayed? 

Maybe some usage example here to give more color, otherwise I don't think users will understand how to use this options

Reply:
If you are dealing with 3d data you'll need to know the visible bounds at a particular elevation. The bounds returned are calculated as the intersection between the frustum and the Z=z plane.

Reply:
But 3D data is not at a particular elevation unless it is a "flat floating plane at elevation z" which seems like a "rather special case". In my mind, 3D  data like terrain tends to be at a wide range of elevations.

That is why I think some color could be useful, something like:

```
To help with calculating bounding volumes for 3D data it is possible to get an approximate estimation of the bounding area by adding a z value representing the average elevation. This is e.g. useful for terrain tiles where each tile tends to be at a fairly common average elevation.
```
Assuming that is indeed the use case.

Reply:
In the TerrainLayer we calculate bounding boxes at minZ and maxZ, and then combine them.